swift/ZKSyncSDK/ZKSyncSDK/ZKSyncSDK.swift

```diff
@@ -0,0 +1,136 @@
+//
+//  ZksCrypto.swift
+//  ZKSyncSDK-UI
+//
+//  Made with ❤️ by Matter Labs on 10/23/20
+//
+
+import Foundation
+import ZKSyncSDK.ZKCryptoKit
+
+/**
+* Access to the ZksCrypto native library
+*
+* This class provides methods for interaction the ZksCrypto native library.
+*
+*/
+public class ZKSyncSDK: NSObject {
+    
+    static public func generatePrivateKey(seed: Data) -> ZKSyncSDKResult<ZKPrivateKey> {
+        return seed.withUnsafeBytes { seedBufferRaw in
+            let seedBuffer: UnsafePointer<UInt8> = seedBufferRaw.baseAddress!.assumingMemoryBound(to: UInt8.self)
+            let result = UnsafeMutablePointer<ZksPrivateKey>.allocate(capacity: 1)
```

Comment:
`ZksPrivateKey` / `ZKPrivateKey` are *very* easy to confuse. Maybe it'll be better to provide more clear names for C structures, e.g. `CReprZKPrivateKey`?

With such an approach, at one hand the type names are will be directly mirrored (without Zks <=> ZK prefix difference), and on the other one both kinds will be easily distinguishable.